CONTRIBUTING.md

```diff
@@ -1,18 +1,26 @@
 # Contributing to Dogecoin Core
```

Comment:
A menu with links at the top could be interesting to facilitate navigation and have a brief overview of available info.

Reply:
Inclined to get the base content updated, and any follow-on thrashing out of content done, then build a TOC (unless there's an automated tool for this, that I'm missing?)